package.json

```diff
@@ -7,6 +7,7 @@
     "build": "webpack --config ./etc/webpack/webpack.dev.config.js",
     "build:prod": "webpack --config ./etc/webpack/webpack.prod.config.js",
     "start": "webpack-dev-server --config ./etc/webpack/webpack-storybook.dev.config.js",
+    "lint": "eslint .",
```

Comment:
During the linting execution seems to have some error at `webpack-storybook.dev.config.js`